tests/features/splash_screen.feature

```diff
@@ -44,9 +44,26 @@ Feature: Splash Screen features
     And I fill in "edit-splash-screen-blacklist" with "fr bg"
     And I press the "Save" button
     Then I should see the success message "The configuration options have been saved."
-    When I go to "/"
+    When I go to "splash"
     Then I should see an "body.not-front.page-splash" element
     And I should see the link "English"
     And I should see the link "Deutsch"
     And I should not see "Български"
     And I should not see "Français"
+
+  @api	
+  Scenario: Administrators can enable/disable the homepage hijacking
+    Given I am logged in as a user with the 'administrator' role
+    When I go to "admin/config/regional/splash_screen_settings"
+    And I check the box "edit-splash-screen-homepage-hijacking"
+    And I press the "Save" button
+    Then I should see the success message "The configuration options have been saved."
+    When I go to "/"
+    Then I should see an "body.not-front.page-splash" element
+    When I go to "admin/config/regional/splash_screen_settings"
+    And I uncheck the box "edit-splash-screen-homepage-hijacking"
+    And I press the "Save" button
+    Then I should see the success message "The configuration options have been saved."
+    When I go to "/"
+    Then I should see an "body.front" element
+    And I should see the heading "Welcome to NextEuropa"
```

Comment:
Behat tests doesn't work.
See continuousphp.
In my dev environment, I have the following message : 
"  [Behat\Gherkin\Exception\ParserException]
  Expected Feature, but got text: "Feature: Splash Screen features" in file: /ec/dev/server/fpfis/webroot/sources/senseca/platform-dev/tests/features/splash_screen.feature"